### PR TITLE
fix: documentation updates

### PR DESCRIPTION
**Pull request by dackers86**
- [x] Updated return PDF description
- [x] Removes boolean value from  displaying in the documentation
- [x] Fixed Typos
- [x] Fixed formatting errors
- [x] Updated security rules descriptions (formatting)

### CODE REVIEW

File: pdf-generator/POSTINSTALL.md [hunk 1]
```diff
@@ -16,24 +17,26 @@ Update your Firebase Storage security rules to allow reads from template path:
...
+      // Allow access to the generated pdf file path
       match /{outputs_path} {
         allow write;
       }
     }
 
 Check [here](https://firebase.google.com/docs/storage/security) for more information about security rules.
 
-***
+---
```
Comment (sassanh) [inline pdf-generator/POSTINSTALL.md:28]: My markdown linter [remark](https://github.com/remarkjs/remark) keeps changing `---` to `***`, is it important to use `---`? If yes I should be able to find a way to configure remark.

File: pdf-generator/POSTINSTALL.md [hunk 2]
```diff
@@ -16,24 +17,26 @@ Update your Firebase Storage security rules to allow reads from template path:
...
 
 ## Executing the function
 
 You can start testing this extension right away.
 
-1.  Create a template based on this [guide](https://github.com/sassanh/template-to-pdf/blob/main/pdf-generator/PREINSTALL.md). Make note of the Handlebars.js parameters you've used in the template, as they should match the request you create in the next step.
```
Comment (sassanh) [inline pdf-generator/POSTINSTALL.md:33]: Remark automatically changes 1, 1, 1 sequence to 1, 2, 3. (I prefer 1, 1, 1)
Which formatter are you using? I will try to use the same thing.

**Review by sassanh — APPROVED** [review]
Approved with 2 questions.

File: pdf-generator/POSTINSTALL.md (hunk 2 above)
Comment (dackers86) [reply pdf-generator/POSTINSTALL.md:33]: Ah ok. We are actively improving our guidelines and documentation.

I've been using [markdown](https://marketplace.visualstudio.com/items?itemName=DavidAnson.vscode-markdownlint) in VsCode.

Open to any suggestions and recommendations if this improved the documentation standard.

File: pdf-generator/POSTINSTALL.md (hunk 1 above)
Comment (dackers86) [reply pdf-generator/POSTINSTALL.md:28]: Is this still an issue, hopefully this configurable as it seems fairly consistent to have *** in most of Extensions I have worked on so far?

Comment (sassanh) [reply pdf-generator/POSTINSTALL.md:28]: Seems like remark is very customizable, it has a configuration mechanism like eslint rules: https://github.com/remarkjs/remark-lint#rules
So all should be fine.

File: pdf-generator/POSTINSTALL.md (hunk 2 above)
Comment (sassanh) [reply pdf-generator/POSTINSTALL.md:33]: Thanks! I think the documentation standard is fine, I just needed to setup my formatter.

**Pull request merged**